src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(({ id }) => id === selectedTabId);
+  const isSelectedTabExists = tabs
+    .find(({ id }) => id === selectedTabId) !== undefined;
```

Comment:
Method 'some' would fit better than 'find'. 

Here you can make the first tab active if isSelectedTabExists === false.